libraries/dotnet/EnergyOrigin.IntegrationEvents/EnergyOrigin.IntegrationEvents.Tests/Events/OrgAcceptedTerms.cs

```diff
@@ -9,9 +9,11 @@ public void can_set_values()
     {
         var subjectId = Guid.NewGuid();
         var tin = "1234567890";
-        var orgAcceptedTerms = new OrgAcceptedTerms(subjectId, tin);
+        var actor = Guid.NewGuid();
```

Comment:
Have you thought about versioining of this event? I'm thinking something like putting it in a V1 folder. So it's Events/OrgAcceptedTerms/V1